scss/2-tools/__tests__/grid.test.js

```diff
@@ -0,0 +1,44 @@
+/* eslint-env jest */
+const sass = require('sass');
+
+describe('cads-grid-col-width', () => {
+  test.each([
+    [1, '8.3333333333%'],
+    [2, '16.6666666667%'],
+    [3, '25%'],
+    [4, '33.3333333333%'],
+    [5, '41.6666666667%'],
+    [6, '50%'],
+    [7, '58.3333333333%'],
+    [8, '66.6666666667%'],
+    [9, '75%'],
+    [10, '83.3333333333%'],
+    [11, '91.6666666667%'],
+    [12, '100%'],
+  ])('outputs width for %i columns', (cols, expected) => {
+    const data = `
+      @use 'scss/2-tools/grid';
+
+      .foo {
+        width: grid.cads-grid-col-width(${cols});
+      }
+    `;
+
+    const result = sass.renderSync({ data });
```

Comment:
`renderSync` is technically deprecated in favour of a new `compileString` API, couldn't work out how to get it to respect `@use` statements so have kept this the same as our existing tests for now and then raised https://github.com/citizensadvice/design-system/issues/2311 to look at this properly.